### PR TITLE
Move answers to fields object

### PR DESCRIPTION
**Pull request by garethmdavies**

### CODE REVIEW

File: server/routes/createCim.test.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const confirmAndSendEnabledHtml =
 
 const confirmRiskBandDisabled =
   '        <div class="govuk-grid-column-two-thirds">' +
-  '\n            <h2 class="heading-medium">Confirm risk band and needs prioritisation</h2>' +
```
Comment (garethmdavies) [inline server/routes/createCim.test.js]: Text change to remove text that is no longer required

File: server/services/formProcessing.js [hunk 2]
```diff
@@ -25,23 +24,32 @@ const getFieldInfo = ({ field, userInput }) => {
   }
 }
 
+const addFieldQuestion = (field, fieldInput) => {
```
Comment (garethmdavies) [inline server/services/formProcessing.js]: New method that is called by each sanitiser to add the question associated with the field.

File: server/utils/fieldValidation.js [hunk 3]
```diff
@@ -40,31 +40,11 @@ const fieldOptions = field => {
       .object()
       .keys({
         answer: joi.valid(['YES', 'NO']).required(),
+        answerLabel: joi.any().optional(),
         question: joi.any().optional(),
       })
       .required(),
```
Comment (garethmdavies) [inline server/utils/fieldValidation.js]: Validator no longer needed

File: server/services/formProcessing.js [hunk 4]
```diff
@@ -25,23 +24,32 @@ const getFieldInfo = ({ field, userInput }) => {
...
+  const updatedFieldInput = fieldInput
```
Comment (yasinmustafa) [inline server/services/formProcessing.js]: This method actually changes the passed in param fieldInput -by the looks of it that wasn't the intention? it looks like you are trying to create a copy, modify it and return the new version whilst leaving the source object unchanged, in which case the following would work:

```
const addFieldQuestion = (field, fieldInput) => {
    const updatedFieldInput = Object.assign({}, fieldInput)
    if (field.text) updatedFieldInput.question = field.text
    return updatedFieldInput
}

```

**Pull request merged**